packages/integrations/react/client.js

```diff
@@ -58,12 +53,13 @@ export default (element) =>
 		for (const [key, value] of Object.entries(slotted)) {
 			props[key] = createElement(StaticHtml, { value, name: key });
 		}
-
-		const componentEl = createElement(
-			Component,
-			props,
-			getChildren(children, element.hasAttribute('data-react-children'))
-		);
+		const componentEl = (
+			  createElement(
```

Comment:
Why is that wrapped inside parenthesis?

Reply:
Good catch, I think I accidentally left it after testing something else. I've added a commit to remove the parentheses.